posthog/session_recordings/session_recording_api.py

```diff
@@ -273,12 +273,13 @@ def _snapshots_v2(self, request: request.Request):
 
         if not source:
             sources: List[dict] = []
-            blob_prefix = recording.build_blob_ingestion_storage_path()
-            blob_keys = object_storage.list_objects(blob_prefix)
 
-            if not blob_keys and recording.storage_version == "2023-08-01":
+            if recording.object_storage_path and recording.storage_version == "2023-08-01":
```

Comment:
this leaves us broken for older version files but we're following up with that. those files need processing anyway before they can be used